dashboard/app/views/plc/index.html.haml

```diff
@@ -7,7 +7,7 @@
 %ul
   %li
     =link_to 'Build a Professional Learning Script', new_script_path
-    (Add "professional_learning_course 'true'" to the script)
+    (Add "professional_learning_course '<course name>'" to the script)
```

Comment:
Ah it looks like `<course name>` isn't getting escaped:

![screen shot 2016-06-28 at 5 03 57 pm](https://cloud.githubusercontent.com/assets/413693/16436396/6215e152-3d52-11e6-86c9-4d36617029e9.png)

+cc @mehalshah